staging/src/k8s.io/client-go/tools/cache/shared_informer.go

```diff
@@ -393,11 +393,7 @@ func WaitForCacheSync(stopCh <-chan struct{}, cacheSyncs ...InformerSynced) bool
 			return true, nil
 		},
 		stopCh)
-	if err != nil {
-		return false
-	}
-
-	return true
+	return err == nil
```

Comment:
we avoid these kind of changes that are noops, also it simplify in the number of lines but not in the readability, this is always a source of discussions 🤷  better avoid this change

Reply:
@aojea Agreed, the IDE gave an S1008 warning, so I submitted this PR.

S1008: Simplify returning boolean expressions

Reply:
> S1008: Simplify returning boolean expressions

yeah, I remember this discussion came from some linter ... so it is this linter, I see, well, at least I and others don't agree this is simpler as in simple to read, is indeed simple as in lines of code